planemo/galaxy/activity.py

```diff
@@ -368,14 +368,17 @@ def _file_path_to_name(file_path):
 
 
 def execute_rerun(
-    ctx: "PlanemoCliContext", config: "BaseGalaxyConfig", rerunnable: Rerunnable, **kwds
+    ctx: "PlanemoCliContext", config: "BaseGalaxyConfig", rerunnable: Rerunnable, use_cache: bool = True, **kwds
 ) -> "GalaxyBaseRunResponse":
     rerun_successful = True
     user_gi = config.user_gi
     if rerunnable.rerunnable_type == "history":
         job_ids = [job["id"] for job in user_gi.jobs.get_jobs(history_id=rerunnable.rerunnable_id, state="error")]
     elif rerunnable.rerunnable_type == "invocation":
-        job_ids = [job["id"] for job in user_gi.jobs.get_jobs(invocation_id=rerunnable.rerunnable_id, state="error")]
+        request = user_gi.invocations._get(f"{rerunnable.rerunnable_id}/request")
+        request["use_cached_job"] = use_cache
+        invocation = user_gi.workflows._post(id=f"{request['workflow_id']}/invocations", payload=request)
```

Comment:
This feels like abusing the `id` parameter, I'd prefer:

```suggestion
        url = "/".join((user_gi.url, "workflows", request["workflow_id"], "invocations"))
        invocation = user_gi.workflows._post(url=url, payload=request)
```

At least, until we have https://github.com/galaxyproject/bioblend/issues/518 fixed on BioBlend side.